src/adapters/aol.js

```diff
@@ -69,7 +56,7 @@ var AolAdapter = function AolAdapter() {
    * @param {ADTECHContext} context the context passed from aol
    */
   function _addBid(response, context) {
-    var bid = bidsMap[context.placement];
+    var bid = bidmanager.getPlacementIdByCBIdentifer(context.alias);
```

Comment:
Yes, the name is definitely confusing.  I guess there's no advantage over a map within this module.  I can update the PR if that makes more sense.  I was trying to figure out what was the standard approach for adapters, and was looking at the appnexus plugin for reference.